src/aiod/default/authorization_test.py

```diff
@@ -1,18 +1,34 @@
+import http.client
+from collections import NamedTuple
+from typing import Sequence
 import requests
 
 from aiod.config.settings import api_base_url
 
 
-def test() -> requests.Response:
-    """
-    Send a test request to the authorization_test endpoint.
+class User(NamedTuple):
+    name: str
+    roles: Sequence[str, ...]
```

Comment:
`Pylance` complains that `...` is not allowed in this context.
```suggestion
    roles: Sequence[str]
```

Reply:
Yes I think this was changed from `Tuple[str, ...]`, must have missed the warning myself.